app/controllers/concerns/current_user.rb

```diff
@@ -66,7 +66,12 @@ def resource_action
   end
 
   def can?(action, resource_namespace, scope = nil)
-    scope ||= current_project if respond_to?(:current_project)
+    case resource_namespace
```

Comment:
this should be `scope ||=` to avoid duplication and to avoid the `respond_to?` calls